boards/x86/acrn/README.rst

```diff
@@ -0,0 +1,230 @@
+Building and Running Zephyr with ACRN
```

Comment:
This needs to go into `boards/x86/acrn/doc/` and either expand on the existing doc or replace it.